.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>ICS372-Assignment1</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
```

Comment:
The Vehicle file looks good. Just get rid of the .project file and I'll approve it

Reply:
resolved, please push in main, thanks!